src/lips.js

```diff
@@ -6920,11 +6920,11 @@ function interaction(env, name) {
 // -------------------------------------------------------------------------
 var internal_env = new Environment({
     stdout: new BufferedOutputPort(function(...args) {
-        console.log(...args);
+        console.log(...args); // @jcubic can you enter what this console log returns here
     }),
     // ------------------------------------------------------------------
     stderr: new BufferedOutputPort(function(...args) {
-        console.error(...args);
+        console.error(...args); // @jcubic can you enter what this console log returns here
```

Comment:
The same but with error output in the Scheme code.

Reply:
Should be deleted now. I made a new commit.